allure-pytest/src/allure/utils.py

```diff
@@ -84,8 +84,8 @@ def allure_full_name(nodeid):
 
 
 def step_parameters(func, *a, **kw):
-    all_names = inspect.getargspec(func).args
-    defaults = inspect.getargspec(func).defaults
+    all_names = inspect.getfullargspec(func).args
```

Comment:
yeah, now I check version with python<3
`Deprecated since version 3.0: Use getfullargspec() for an updated API that is usually a drop-in replacement, but also correctly handles function annotations and keyword-only parameters.`